praveglet/README.md

```diff
@@ -0,0 +1,47 @@
+
+# Praveglet
+
+Praveglet is an app that can be used to interact with Pravega. This utility can perform the following operations
+* Create a Scope
+* Create a Stream
+* Write events into a Stream from STDIN or FIFO.
```

Comment:
The code as-written doesn't actually read from a FIFO, only standard in.

Reply:
corrected it.